lib/jsonapi/include_directive.rb

```diff
@@ -12,6 +12,8 @@ module JSONAPI
   # @example 'posts.**' # => Include related posts, and all the included
   #   posts' related resources, and their related resources, recursively.
   class IncludeDirective
+    attr_reader :options, :hash
```

Comment:
Thanks for the advices!
I will only be able to add more commits from the 20th of September as I'm on holiday at the moment 😅